administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -14,6 +16,8 @@ COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_DEFAULT="Default"
 COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_LABEL="Update Channel"
 COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_NEXT="Joomla Next"
 COM_JOOMLAUPDATE_CONFIG_UPDATESOURCE_TESTING="Testing"
+COM_JOOMLAUPDATE_CONFIG_VERSIONCHECK_DESC="Shows the checkbox in the pre–update check, displayed when upgrading to a new Joomla version family (minor or major version), if any of the extensions installed on your site is potentially incompatible with the version of Joomla you are upgrading to."
```

```suggestion
COM_JOOMLAUPDATE_CONFIG_VERSIONCHECK_DESC="Shows the checkbox in the pre–update check if any of the extensions installed on your site is potentially incompatible with the version of Joomla you are upgrading to. Note: the checkbox is displayed when upgrading to a new Joomla version family (minor or major version),"
```

slightly change the wording to simplify it by splitting into two sentences.